Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -205,6 +204,7 @@ PoolMultiThreader ::ParallelizeArray(SizeValueType             firstIndex,
       {
         filter->UpdateProgress(i / float(workUnit));
       }
+      // TODO: Wrap in exception handling
```

Comment:
I don't think that parallelize array can be aborted. Also, I think we want the exception to be propagated to the caller in its unaltered form.

Reply:
But the `.get` call may throw an exception, when other threads are still running.

Reply:
It should not be hard to record the exception and re-throw it after the for loop, in a way similar the the rest of this patch?

Reply:
OK I am writing a follow-up commit for this.